ranger/gui/widgets/taskview.py

```diff
@@ -80,6 +80,11 @@ def task_remove(self, i=None):
         if self.fm.loader.queue:
             self.fm.loader.remove(index=i)
 
+    def task_remove_all(self):
+        if self.fm.loader.queue:
```

Comment:
I think you can use something like:
```
def task_remove_all(self):
    for i in self.fm.loader.queue:
        task_remove(i)
```

Reply:
So rather than mapping dD to `fm.loader.destroy()`, I kept it in the newly created `task_remove_all()` function so that the intent is clearer.